lib/controllers/authors.js

```diff
@@ -0,0 +1,14 @@
+const { Router } = require('express');
+const Author = require('../models/Author');
+
+module.exports = Router()
+  .get('/', async (req, res) => {
+    const authors = await Author.getAll();
+    const ids = authors.map((author) => ({
```

Comment:
Same here, there should be no need for a map, because in your `SELECT` you grab the id and name straight from the table